docker/build-container.sh

```diff
@@ -3,12 +3,37 @@
 export UID=$(id -u)
 export GID=$(id -g)
 
-for arg in $@; do
-    if [[ $arg == clean ]]; then
-        docker rm cv-hardware-parts
-	docker rmi ia-bigdata
-    fi;
+while [[ $# -gt 0 ]]; do
+    case $1 in 
+    --clean-container | --clean-all)
+        docker rm ai-palatal-rugoscopy-env > /dev/null 2>&1
+        ;;
+    --clean-image | --clean-all)
+        docker rmi ai-palatal-rugoscopy > /dev/null 2>&1
+        ;;
+    --data-path)
+        data_path=$2
+    esac
+    shift
 done
 
-docker build --build-arg USER=$USER --build-arg PW=cv123 --build-arg UID=$UID --build-arg GID=$GID -t ia-bigdata .
-docker run -it --gpus all -v $(dirname `pwd`):/home/$USER/workspace/cv-hardware-parts -v $HOME/SMACS-0723:/home/$USER/server --name cv-hardware-parts ia-bigdata
+# Check whether path to data folder is provided by the user, otherwise no data will be available inside the container.
+if [[ -z $data_path ]]; then
+    echo "ERROR: Data folder not specified"
+    exit 1
+fi
+
+docker build \
+    --build-arg USER=$USER \
+    --build-arg PW=88452452 \
```

Comment:
Is this a password or something?

Reply:
Yes. I wasn't so inspired when creating this one :laughing:. 
I'm not sure if we really need this, since a default password is already being defined in the Dockerfile.

Reply:
Just tested without passing the password and it works. I will remove it from the shell script.